test/common/pixel-tests

```diff
@@ -33,6 +38,20 @@ cmd_status() {
 }
 
 cmd_push() {
+    # Check if the current branch is behind the project's main and has newer pixel-tests, in this case
+    # the user has to rebase to not run into pixel-tests conflicts. There is no
+    # elegant way figuring out the real "origin" of the cockpit project so we
+    # discover the project's repository name and hardcode the upstream url.
+    project=$(git remote -v | awk '/^origin/ { print $2; exit }' | xargs basename)
```

Comment:
Does `git remote get-url origin` work? That would be less magic.

We have some code [here](https://github.com/cockpit-project/bots/blob/main/task/github.py#L170)
 to figure out the repo. We should probably use it in this PR and also in #17422.